website/docs/r/licensemanager_association.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_licensemanager_association"
+sidebar_current: "docs-aws-resource-licensemanager-license-configuration"
```

Comment:
🍝 Copy-pasta typo and missing sidebar update in `website/aws.erb`

```suggestion
sidebar_current: "docs-aws-resource-licensemanager-association"
```

Reply:
Doh 🍩